content/first-steps.md

```diff
@@ -112,3 +112,49 @@ $ npm run start:dev
 ```
 
 This command will watch your files, automatically recompiling and reloading the server.
+
+#### Linting / formatting
```

Comment:
```suggestion
#### Linting and formatting
```
Otherwise looks good, only the title I would say remove the "/" also to be in line with the titles of the documentation in general.